src/frontends/pytorch/src/transforms/listconstruct_replacer.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "listconstruct_replacer.hpp"
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/util/framework_node.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/roll.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/unsqueeze.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/or.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+using namespace ov::pass;
+using namespace ov::op;
+
+ListConstructReplacer::ListConstructReplacer() {
+    // Transformation for torch operators for cases where prim::ListConstruct can be replaced with Concat.
+    auto list_construct = pattern::wrap_type<ov::op::util::FrameworkNode>();
+
+    // Both aten::view and aten::reshape are using same translation returning Reshape operator.
+    auto reshape_op = pattern::wrap_type<v1::Reshape>({pattern::any_input(), list_construct});
+    //auto roll_op = pattern::wrap_type<v7::Roll>({pattern::any_input(), list_construct, pattern::any_input()});
```

Comment:
does it some debug comments and should be removed?

Reply:
Good catch, yeah, that is artifacts from testing